SPECS-EXTENDED/pykickstart/pykickstart.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+ "Signatures": {
+  "pykickstart-3.36.tar.gz": "2721fe2cdc3ae6e485f2a3344c555d6ed505b412d3dd19c1d00492a106e60b98",
+  "pykickstart-3.36.tar.gz.asc": "ef1ff421210a110dd1e11d50d0770c9333b0b51959d30507a7a50fb3bed47097"
```

Comment:
This line is a vestege of our previous removals :)

Reply:
I have removed this as well. Please check the latest commit.